.github/workflows/release.yml

```diff
@@ -13,18 +13,35 @@ jobs:
   snapshot:
     name: Save a snapshot
     if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
+    runs-on: macos-13
```

Comment:
🤔 might want to try `macos-latest` - [an M1 machine](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) - once more